### PR TITLE
feat(ci): add rule get_clinical_guidelines

### PR DESCRIPTION
**Pull request by chels0**
### This PR:

(If this is a release PR, no need to add following. Leave this part empty)
(Use the following lines to create a PR text body. Make sure to remove all non-relevant one after you're done)
(Repeat each field as many times as necessary)

Added: for new features.
Changed: for changes in existing functionality.
Deprecated: for soon-to-be removed features.
Removed: for now removed features.
Fixed: for any bug fixes.
Security: in case of vulnerabilities.

### Review and tests:
- [ ] Tests pass
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Code review
- [ ] `CHANGELOG.md` is updated
- [ ] New code is executed and covered by tests, and test approve
- [ ] Module compatibility section in `README.md` is up to date

### CODE REVIEW

File: .tests/integration/reference/generate_pgx_report.Rmd
Comment (jonca79) [inline .tests/integration/reference/generate_pgx_report.Rmd:1]: remove file

File: workflow/rules/get_clinical_guidelines.smk [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        mem_per_cpu=config.get("get_clinical_guidelines", {}).get("mem_per_cpu", config["default_resources"]["mem_per_cpu"]),
+        partition=config.get("get_clinical_guidelines", {}).get("partition", config["default_resources"]["partition"]),
+        threads=config.get("get_clinical_guidelines", {}).get("threads", config["default_resources"]["threads"]),
+        time=config.get("get_clinical_guidelines", {}).get("time", config["default_resources"]["time"]),
+    container:
+        config.get("get_clinical_guidelines", {}).get("container", config["default_container"])
+    conda:
+        "../envs/get_clinical_guidelines.yaml"
+    message:
+        "{rule}: Given detected variants, get possible Haplotype combinations on {input}"
```
Comment (Smeds) [inline workflow/rules/get_clinical_guidelines.smk:37]:
```suggestion
        "{rule}: given detected variants, get possible Haplotype combinations on {input}"
```

File: workflow/rules/get_interaction_guidelines.smk [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        mem_per_cpu=config.get("get_interaction_guidelines", {}).get("mem_per_cpu", config["default_resources"]["mem_per_cpu"]),
+        partition=config.get("get_interaction_guidelines", {}).get("partition", config["default_resources"]["partition"]),
+        threads=config.get("get_interaction_guidelines", {}).get("threads", config["default_resources"]["threads"]),
+        time=config.get("get_interaction_guidelines", {}).get("time", config["default_resources"]["time"]),
+    container:
+        config.get("get_interaction_guidelines", {}).get("container", config["default_container"])
+    conda:
+        "../envs/get_interaction_guidelines.yaml"
+    message:
+        "{rule}: Given Haplotype Combinations, get possible interactions betweens these on {input}"
```
Comment (Smeds) [inline workflow/rules/get_interaction_guidelines.smk:34]:
```suggestion
        "{rule}: given Haplotype Combinations, get possible interactions betweens these on {input}"
```

File: workflow/scripts/get_clinical_guidelines.py [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+    if not df.empty:
+        hidden_haplotypes = pd.read_csv(hidden_haplotypes, sep="\t")
+        hidden_haplotypes["comb"] = hidden_haplotypes[[
+            "Haplotype1", "Haplotype2"
+        ]].apply(lambda x: "".join(sorted(x.tolist())), axis=1)
+        df["comb"] = df[["Haplotype1", "Haplotype2"
+                         ]].apply(lambda x: "".join(sorted(x.tolist())),
+                                  axis=1)
+        print(df["comb"])
+        print(hidden_haplotypes["comb"])
```
Comment (Smeds) [inline workflow/scripts/get_clinical_guidelines.py:233]: Is this for debugging?

If it's not you should consider using logging https://docs.python.org/3/howto/logging.html

**Review by Smeds — APPROVED** [review]

**Pull request merged**